packages/core/src/model.ts

```diff
@@ -454,6 +454,16 @@ export class ChartModel {
 		return this.colorScale;
 	}
 
+	getColorForGroup(group: any, key?: any, data?: any) {
```

Comment:
We already have a `getFillColor` in model that'll call `options.getFillColor` if it exists